### DISCUSSION TITLE
MAVLink Messages: LOCAL_POSITION_NED

### DISCUSSION THREAD

**Discussion by Cofalito** [bluerobotics/BlueOS#2574] [Q&A] [answered]
Hi, I've been trying to get the MAVLink message LOCAL_POSITION_NED from the BlueROV2. After checking in QGroundControl and BlueOS, I wasn't able to find it. Does anyone know how can I add this message, or any of the common messages?

**Comment by patrickelectric** [comment, marked as answer]
Hi @Cofalito this is an message that you'll only receive if you are using a position system connected with the vehicle.
